editions/tw5.com/tiddlers/widgets/GenesisWidget.tid

```diff
@@ -1,6 +1,7 @@
+bad-example: <$genesis $type="my-element" $names="one two" $values="1 2">Test Genesis Widget</$genesis>
```

Comment:
Why not use the usual `wikitext-example` macro here?

Reply:
will change that